packages/shared/src/testing/playwright.vrt.config.ts

```diff
@@ -1,5 +1,26 @@
 import { type Config } from '@playwright/test';
 
+export const themes = ['light', 'dark'] as const;
+export const schemes = ['light', 'dark'] as const;
+
+export const viewportWidthXXS = 320;
```

Comment:
Might be good to export an object that contains the relevant viewport presets to reduce manual maintenance and importing/concatenating/filtering them in various tests.

Reply:
I'll check.